src/components/Signup/components/container/index.jsx

```diff
@@ -0,0 +1,141 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { Link } from 'react-router-dom';
+import '../../../login/styles/index.css';
+
+export class Signup extends Component {
+  state = {
+    firstname: '',
+    lastname: '',
+    email: '',
+    username: '',
+    password: '',
+    confirmpassword: '',
+  };
+  render() {
+    const {
+      firstname, lastname, email, username, password, confirmpassword,
+    } = this.state;
+    return (
+      <div className="container">
+        <div className="row">
+          <div className="col-md-5 m-auto">
+            <p className="paragraphs">
+              <h1>Open Concept Lab</h1>
+              <h4>for OpenMRS</h4>
+            </p>
+            <p className="paragraphz">
+              Use the shared Open Concept Lab to create OpenMRS dictionaries by mixing
+              expert-defined content with your own custom concepts.
+            </p>
+          </div>
+          <div className="col-md-7 signup">
+            <form>
+              <fieldset>
+                <legend>Sign Up to Open Concept Lab</legend>
+                <div className="form-group">
+                  <p className="paragraph">First Name</p>
+                  <input
+                    type="text"
+                    className="form-control"
+                    placeholder="Enter First name"
+                    name="firstname"
+                    id="firstname"
+                    required
+                    value={firstname}
+                    onChange={this.handleInput}
+                  />
+                </div>
+                <div className="form-group">
+                  <p className="paragraph">Last Name</p>
+                  <input
+                    type="text"
+                    className="form-control"
+                    placeholder="Enter Last name"
+                    name="lastname"
+                    id="lastname"
+                    required
+                    value={lastname}
+                    onChange={this.handleInput}
+                  />
+                </div>
+                <div className="form-group">
+                  <p className="paragraph">Email</p>
+                  <input
+                    type="text"
+                    className="form-control"
+                    placeholder="Enter your Email"
+                    name="email"
+                    id="email"
+                    required
+                    value={email}
+                    onChange={this.handleInput}
```

Comment:
I see you are calling a function `handleInput`, where is this function declared?

Reply:
+1 on this.

Reply:
I made the outlined changes. Please review if I have made the changes.